tests/unit/dataSource/properties/tests.html

```diff
@@ -453,7 +532,20 @@
 			equal(dsLocal.pendingTransactions().length, 0, "There should be a no pending transaction after the transaction has been rolled back.");
 		});
 
+		test("Test _removeTransactionsByRecordId API", function () {
+			var transaction = dsLocal.addRow(999, { "ID": 999, "Name": "Bread", "Price": 2.5 }, false);
```

Comment:
You should have more than one transaction pending, so that you can see whether the correct transaction is removed.